nuntium/user_section/views.py

```diff
@@ -331,13 +319,68 @@ def get_redirect_url(self, *args, **kwargs):
             writeitinstance__owner=self.request.user
             )
         message.moderate()
-        view_messages.info(self.request, _("This message has been moderated"))
+        view_messages.info(self.request, _('The message "%(message)s" has been accepted') % { 'message': message })
         return reverse(
             'messages_per_writeitinstance',
             subdomain=message.writeitinstance.slug,
             )
 
 
+class RejectMessageView(RedirectView):
+    permanent = False
+
+    @method_decorator(login_required)
+    def dispatch(self, *args, **kwargs):
+        return super(RejectMessageView, self).dispatch(*args, **kwargs)
+
+    def get_redirect_url(self, *args, **kwargs):
+        message = get_object_or_404(Message,
+            pk=kwargs['pk'],
+            writeitinstance__slug=self.request.subdomain,
+            writeitinstance__owner=self.request.user
+            )
+        message.public = False
+        message.moderated = True
+        message.save()
```

Comment:
I'm thinking that perhaps we could do something with the OutboundMessage related to this message, apparently they should keep on being in status 'new' as [in here](https://github.com/ciudadanointeligente/write-it/pull/859/files#diff-5e559ebeaac71a53dc393dcd11f56404R398), but I think that since we're rejecting the message we could may be set those OutboundMessages to 'error' or something. Or even we could create a new status for them 'rejected'. I'd just be afraid that some of those messages keep around and they could still be sent to the recipient.